client-feature-matrix/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: pulsar-client-feaure-matrix
+title: Pulsar Client Feaure Matrix
+sidebar_label: Pulsar Client Feaure Matrix
+---
+
+import Matrix from "../src/components/Matrix/matrix";
+import {
+  languages,
+  client,
+  producer,
+  reader,
+  consumer,
+  tableview,
+} from "../data/matrix.js";
+
+### Client
+
+<Matrix data={client} name="client" />
+
+### Producer
+
+<Matrix data={producer} name="producer" />
+
+### Consumer
+
+<Matrix data={consumer} name="consumer" />
+
+See details and full version [here](https://docs.google.com/spreadsheets/d/1YHYTkIXR8-Ql103u-IMI18TXLlGStK8uJjDsOOA0T20/edit?usp=sharing)
```

Comment:
After we land this page, shall this page be the source of truth so that we don't refer to the gdoc? Otherwise, why do we add such a page?

Reply:
Yes, it should be the source of truth. However, the page is a compact version at the pilot stage.
In case some users or maitainers still need more information from the working sheet, we attach the link here to not lose the connection with the full version.

Reply:
Weird. Then we create two similar but different matrices, which seems not a good idea.

Reply:
For example, I'm a developer, I cannot trust the page but have to look at the gdoc version because any updates will go there first. As a user, it's the same. This patch introduces a _maybe_ up-to-date version but any updates and nightly changes happen on the gdoc. Then I suggest we use only the gdoc and that should not worse the case.

Reply:
Maybe I didn't make it clear. In this PR, we only add the matrix for Client, Producer, and Consumer. With more feedback/voice from the community, we will iterate and continue to add the matrix for Reader and TableView. After that, we can move the link.

Reply:
Thanks for your explanation! I'd prefer to remove the link so that it's clear that we'd like to move to this page. Instead of leaving a workaround and it's highly possible the intermediate situation remains for a long time.